packages/neos-ui-editors/src/TextArea/index.js

```diff
@@ -9,14 +9,16 @@ const defaultOptions = {
 };
 
 const TextAreaEditor = props => {
-    const {value, commit, highlight, options} = props;
+    const {value, commit, highlight, options, validationErrors} = props;
 
     const finalOptions = Object.assign({}, defaultOptions, options);
 
-    return (<TextArea
+    return (<TextAreaWithValidationResultRendering
         value={value}
         onChange={commit}
         highlight={highlight}
+        invalid={validationErrors && validationErrors.length > 0}
```

Comment:
I'd call this property "isValid"; thinking about double-negated properties gives me some headaches ;)

Reply:
Hm, the textarea as well as the textfield component already got the invalid prop, should i change those also?

Reply:
ah, I'd change them. but no blocker :)